### ISSUE TITLE
Use arkit in landscape mode

### ISSUE THREAD

**Issue by Ezor** [zac/iphonearkit#1]
Hello

I have tried to modify the code to use arkit in landscape mode but it didn't work. Can you provide some code (maybe by adding some comments in the source code ?) to explain how to use arkit in landscape mode.

Anyway thanks for this usefull project.

Julien "Ezor"

**Comment by delagroove** [comment]
- (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation {
  // Return YES for supported orientations
  return (interfaceOrientation == UIInterfaceOrientationLandscapeLeft || interfaceOrientation == UIInterfaceOrientationLandscapeRight);
  }

**Comment by JustinKent** [comment]
I don't think this alone will work... a lot more needs to be done to support horizontal orientation. Try using this fork instead: 
http://github.com/nielswh/iPhone-AR-Toolkit/commit/54dd015bd6a5a34e2e5152667e855ed01a133047
I haven't tried it, but it should work.

**Comment by nielswh** [comment]
I've added this feature to my fork.  It was a bit more work that just setting the orientation.  I had to  also adjust the compass setting and other methods to make it work.
Check out  https://github.com/nielswh/iPhone-AR-Toolkit for more information.
